features/steps/app_steps.rb

```diff
@@ -3,8 +3,8 @@
 end
 
 When('I relaunch the app') do
-  case Maze.driver.capabilities['platformName']
-  when 'Mac'
+  case Maze::Helper.get_current_platform
+  when 'macos'
```

Comment:
Would it be better if these were symbols (i.e. `:macos`) instead of strings?

Reply:
Yes, almost certainly - I realised that too late, after releasing Maze Runner 😞 .  We could fix that in the future.